x-pack/plugins/stack_connectors/server/connector_types/stack/torq/index.ts

```diff
@@ -0,0 +1,378 @@
+  const token = secrets.token;
+
+  const axiosInstance = axios.create();
+  const result: Result<AxiosResponse, AxiosError<{ message: string }>> = await promiseResult(
+    request({
+      axios: axiosInstance,
+      url: webhookIntegrationUrl,
+      method: 'post',
+      headers: {
+        'X-Torq-Token': token || '',
+        'Content-Type': 'application/json',
+      },
+      data: JSON.parse(data || 'null'),
+      configurationUtilities,
+      logger: execOptions.logger,
+      validateStatus: (status: number) => status >= 200 && status < 300,
+    })
+  );
+
+  if (isOk(result)) {
+    const {
+      value: { status, statusText },
+    } = result;
+    execOptions.logger.debug(
+      `response from Torq action "${actionId}": [HTTP ${status}] ${statusText}`
+    );
+    return successResult(actionId, data);
+  }
+  const { error } = result;
+  return handleExecutionError(error, execOptions.logger, actionId);
+}
+
+async function handleExecutionError(
+  error: AxiosError<{ message: string }>,
+  logger: Logger,
+  actionId: string
+): Promise<ActionTypeExecutorResult<unknown>> {
+  if (error.response) {
+    const {
+      status,
+      statusText,
+      headers: responseHeaders,
+      data: { message: responseMessage },
+    } = error.response;
+    const responseMessageAsSuffix = responseMessage ? `: ${responseMessage}` : '';
+    const message = `[${status}] ${statusText}${responseMessageAsSuffix}`;
+    logger.error(`error on ${actionId} Torq event: ${message}`);
+    // The request was made and the server responded with a status code
+    // that falls out of the range of 2xx
+    // special handling for 5xx
+    if (status >= 500) {
+      return retryResult(actionId, message);
+    }
+
+    // special handling for rate limiting
+    if (status === 429) {
+      return pipe(
+        getRetryAfterIntervalFromHeaders(responseHeaders),
+        map((retry) => retryResultSeconds(actionId, message, retry)),
+        getOrElse(() => retryResult(actionId, message))
+      );
+    }
+
+    if (status === 405) {
+      return errorResultInvalidMethod(actionId, message);
+    }
+
+    if (status === 401) {
+      return errorResultUnauthorised(actionId, message);
+    }
+
+    if (status === 404) {
+      return errorNotFound(actionId, message);
+    }
+
+    return errorResultInvalid(actionId, message);
+  } else if (error.code) {
+    const message = `[${error.code}] ${error.message}`;
+    logger.error(`error on ${actionId} Torq event: ${message}`);
+    return errorResultRequestFailed(actionId, message);
+  } else if (error.isAxiosError) {
+    const message = `${error.message}`;
+    logger.error(`error on ${actionId} Torq event: ${message}`);
+    return errorResultRequestFailed(actionId, message);
+  }
+  logger.error(`error on ${actionId} Torq action: unexpected error`);
+  return errorResultUnexpectedError(actionId);
+}
+
+// Action Executor Result w/ internationalisation
```

Comment:
```suggestion
// Action Executor Result
```

(or remove the comment completely, doesn't seem to add much)